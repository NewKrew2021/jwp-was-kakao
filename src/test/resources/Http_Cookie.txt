HTTP/1.1 302 Found
set-cookie: logined=true; Path=/
location: /index.html

