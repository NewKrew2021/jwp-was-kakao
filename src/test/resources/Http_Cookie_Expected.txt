HTTP/1.1 302 Found
Set-Cookie: logined=true; Path=/
Location: /index.html

